extension/src/pipeline/quickPick.ts

```diff
@@ -1,40 +1,54 @@
 import isEqual from 'lodash.isequal'
+import { QuickPickItemKind } from 'vscode'
 import {
   PLOT_TEMPLATES,
   Value,
   ValueTree,
   isValueTree
 } from '../cli/dvc/contract'
-import { loadDataFile } from '../fileSystem'
-import { quickPickOne } from '../vscode/quickPick'
-import { pickFile } from '../vscode/resourcePicker'
+import { getFileExtension, loadDataFiles } from '../fileSystem'
+import { quickPickOne, quickPickValue } from '../vscode/quickPick'
+import { pickFiles } from '../vscode/resourcePicker'
 import { Title } from '../vscode/title'
 import { Toast } from '../vscode/toast'
+import { getFileName } from '../fileSystem/util'
 
-const pickDataFile = () => {
-  return pickFile(Title.SELECT_PLOT_DATA, {
+const pickDataFiles = (): Thenable<string[] | undefined> =>
+  pickFiles(Title.SELECT_PLOT_DATA, {
     'Data Formats': ['json', 'csv', 'tsv', 'yaml']
   })
-}
 
-const pickTemplateAndFields = async (
-  fields: string[]
-): Promise<{ x: string; y: string; template: string } | undefined> => {
+const pickTemplateAndFields = async (fields: {
+  [file: string]: string[]
+}): Promise<PlotConfigData | undefined> => {
   const template = await quickPickOne(PLOT_TEMPLATES, 'Pick a Plot Template')
 
   if (!template) {
     return
   }
 
-  const x = await quickPickOne(fields, 'Pick a Metric for X')
+  const items = []
+
+  for (const [file, keys] of Object.entries(fields)) {
+    items.push(
+      {
+        kind: QuickPickItemKind.Separator,
+        label: getFileName(file),
```

Comment:
Now looking at this, the filenames could be the same (`train/values.json` and `test/values.json` for example). I'll update it to show the relative path.